frontend/src/slices/gameDataSlice.ts

```diff
@@ -95,9 +95,7 @@ const gameDataSlice = createSlice({
       EventBus.emit("upgrade-purchased", ups.acquired);
     },
     puzzleSolve: (state, action: PayloadAction<string>) => {
```

Comment:
Is there any particular reason why the debugging `console.log()` shouldn't be kept?